packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -133,7 +134,15 @@ export const SelectItem = styled( Ariakit.SelectItem )`
 	justify-content: space-between;
 	padding: ${ ITEM_PADDING };
 	font-size: ${ CONFIG.fontSize };
-	line-height: 2.15rem; // TODO: Remove this in default but keep for back-compat in legacy
+	// TODO: reassess line-height for non-legacy v2
+	line-height: 28px;
```

Comment:
I'd prefer `2.15384615` and adding a comment about how it's calculated, because that way we'll keep the flexibility/relativity in case someone modifies only the `font-size`.

Reply:
See https://github.com/WordPress/gutenberg/pull/62825#discussion_r1661279512